recipes/climtools/25.2/gh200/environments.yaml

```diff
@@ -0,0 +1,24 @@
+gcc-env:
+  compiler:
+      - toolchain: gcc
+        spec: gcc
+  mpi:
+      spec: cray-mpich@8.1.30
+      gpu: false
+  unify: true
+  specs:
+  - hdf5
```

Comment:
`hdf5 +szip +hl`: the `hl` option is required if anybody wants to use the Python HDF5 interface.

Reply:
Thanks for the info, I can add it in case. Not sure anybody directly uses hdf5 from python though. People use the netcdf APIs, which should then interface with hdf5 through C. Anyhow, I guess it doesn't harm.